test/integration/local_transactions_test.rb

```diff
@@ -365,8 +367,8 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
     end
 
     should "show advisory message that we have no url" do
-      assert page.has_content?("We don't have a link for their website.")
-      assert page.has_link?("local council search", href: "/find-local-council")
+      content = "We don't have a link for their website. Try the &lt;a href=\"/find-local-council\"&gt;local council search&lt;/a&gt; instead."
```

Comment:
Oh you're actually asserting that plain HTML is output here rather than actual HTML. It should be asserting against actual HTML - or perhaps better to do an I18n lookup.


(if you get two very similar comments on this it's because I wrote one and submitted it and GitHub no longer seems to have it)

Reply:
👍